py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py

```diff
@@ -263,22 +244,21 @@ def aten_ops_embedding(
         name,
         input=args[1],
         weight=args[0],
-        # args[2] is the padding index, which is useful for training only
-        scale_grad_by_freq=args_bounds_check(args, 3),
-        sparse=args_bounds_check(args, 4),
+        # args[2, 3, 4] are useful for training only
+        padding_idx=args_bounds_check(args, 2, -1),
+        scale_grad_by_freq=args_bounds_check(args, 3, False),
+        sparse=args_bounds_check(args, 4, False),
     )
 
 
 def embedding_bag_validator(node: Node) -> bool:
-    mode = args_bounds_check(node.args, 4, 0)
-    indices = node.args[1].meta.get("tensor_meta")
+    meta = node.args[1].meta
+    indices = meta.get("tensor_meta")
+    if indices is None:
+        indices = meta.get("val")
     if indices is None:
         return False
-    return (
-        bool(node.args[2].op == "get_attr")
-        and (mode == 0 or mode == 1 or mode == 2)
-        and len(indices.shape) == 1
-    )
```

Comment:
I believe this validator still needs to verify that none of the downstream users access the items at any indices other than index 1. Essentially, it needs the code from here:
https://github.com/pytorch/TensorRT/blob/7f14221ca532d8c42daa43b74fe600ebb27e1537/py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py#L48-L54